includes/class-sensei-admin.php

```diff
@@ -634,29 +634,71 @@ private function duplicate_lesson_quizzes( $old_lesson_id, $new_lesson_id ) {
 		}
 	}
 
+	/**
+	 * Update prerequisite ids after course duplication
+	 *
+	 * @param  array $lessons_to_update    List with lesson_id and old_prerequisite_id id to update.
+	 * @param  array $new_lesson_id_lookup History with the id before and after duplication.
+	 * @return void
+	 */
+	private function update_lesson_prerequisite_ids( $lessons_to_update, $new_lesson_id_lookup ) {
+		foreach ( $lessons_to_update as $lesson_to_update ) {
+			$old_prerequisite_id = $lesson_to_update['old_prerequisite_id'];
+			$new_prerequisite_id = $new_lesson_id_lookup[ $old_prerequisite_id ];
+			add_post_meta( $lesson_to_update['lesson_id'], '_lesson_prerequisite', $new_prerequisite_id );
+		}
+	}
+
+	/**
+	 * Get an prerequisite update object
+	 *
+	 * @param  integer $old_lesson_id ID of the lesson before the duplication.
+	 * @param  integer $new_lesson_id New ID of the lesson.
+	 * @return array                  Object with the id of the lesson to update and its old prerequisite id.
+	 */
+	private function get_prerequisite_update_object( $old_lesson_id, $new_lesson_id ) {
+		$lesson_prerequisite = get_post_meta( $old_lesson_id, '_lesson_prerequisite', true );
+		if ( ! empty( $lesson_prerequisite ) ) {
```

Comment:
We lean towards putting a blank line before and after control structures like `if` statements and loops for improved readability.

Reply:
Perfect! I'll follow it on the next codes! Thank you for the tip!